src/lib/y2storage/boot_requirements_strategies/legacy.rb

```diff
@@ -183,7 +188,8 @@ def errors_on_msdos
       def unknown_boot_partition_table_error
         # TRANSLATORS: error message
         error_message = _(
-          "Boot disk is without partition table"
+          "Boot disk is without partition table and we does not support " \
+            "booting from it. You can fix it by creating partition table on disk."
```

Comment:
NP: I will improve the text a little bit more.

"Boot disk has no partition table and it is not possible to boot from it." \
"You can fix it by creating a partition table on the boot disk."